packages/augur-core/tests/reporting/test_reporting.py

```diff
@@ -141,9 +144,9 @@ def test_initialReport_methods(localFixture, universe, market, constants):
 
 @mark.parametrize('rounds', [
     2,
-    3,
-    6,
-    16
+    #3,
```

Comment:
Any reason to leave these commented-out instead of deleting them?

Reply:
fixed